data/settings/icon-grid-C.json.in

```diff
@@ -3,24 +3,41 @@
     "chromium-browser.desktop",
     "org.gnome.Nautilus.desktop",
     "libreoffice-writer.desktop",
-    "rhythmbox.desktop",
-    "com.endlessm.photos.desktop",
+    "libreoffice-calc.desktop",
+    "libreoffice-impress.desktop",
     "com.endlessm.encyclopedia.en.desktop",
-    "net.minetest.Minetest.desktop",
+    "eos-link-facebook.desktop",
+    "eos-link-whatsapp.desktop",
     "eos-link-youtube.desktop",
     "eos-folder-media.directory",
-    "eos-folder-games.directory",
-    "eos-folder-work.directory",
     "eos-folder-curiosity.directory",
-    "eos-folder-social.directory"
+    "eos-folder-games.directory",
+    "eos-folder-social.directory",
+    "net.minetest.Minetest.desktop",
```

Comment:
Okay, but keep in mind that we typically use the default layout for screenshots on the website and marketing materials, so in some sense how it looks out of the box should have a little more emphasis.  But let's go ahead and see how it looks.